src/components/common/unlock-button/unlock-button.scss

```diff
@@ -17,26 +17,26 @@
   }
 
   .unlock-button-label {
-    @mvx:text-base mvx:leading-none;
-    color: #e8e8e8;
+    @apply mvx:text-base mvx:leading-none;
+    color:  var(--mvx-text-color-primary);
   }
 
   .unlock-button-status {
     @apply mvx:ml-auto mvx:leading-none mvx:flex mvx:items-center mvx:py-1 mvx:px-2 mvx:font-medium mvx:gap-1;
     font-size: 12px;
     border-radius: 20px;
-    border: 1px solid rgba(64, 64, 64, 0.4);
-    background: rgba(11, 11, 11, 0.8);
+    border: 1px solid var(--mvx-border-color-primary);
+    background: var(--mvx-bg-color-primary);
```

Comment:
Same thing as above, we shouldn't hold opacity settings in color variables. The opacity should be set separatedly, we need to change the styles here a little to reflect that.